pid.py

```diff
@@ -1,13 +1,42 @@
 class PID:
-	def __init__(self):
+	def __init__(self, a, b, c, initial_error=None):
+		# (a, b, c) are the weights for (p, i, d)
+		self.a = a
+		self.b = b
+		self.c = c
```

Comment:
Instead of `a`, `b`, and `c` those variables should be named `P`, `I`, and `D` since they are the constants for the PID control scheme.